sdcm/nemesis.py

```diff
@@ -443,7 +443,15 @@ def _terminate_cluster_node(self, node):
         self.cluster.terminate_node(node)
         self.monitoring_set.reconfigure_scylla_monitoring()
 
-    def disrupt_nodetool_decommission(self, add_node=True):
+    def disrupt_nodetool_decommission(self, add_node=True, is_seed_node=False):
+        if is_seed_node:
+            if len(self.cluster.seed_nodes) < 2:
+                raise UnsupportedNemesis("To running seed decommission the cluster must contains at least 2 seed nodes")
+            if not self.target_node.is_seed:
+                self.target_node = random.choice(self.cluster.seed_nodes)
+            seed_address = ",".join([node.ip_address for node in self.cluster.seed_nodes])
+            [node.patch_scylla_yaml(seed_address=seed_address) for node in self.cluster.nodes]
```

Comment:
don't use list comprehension with assignment, it's more readable to use regular for loop here.

Reply:
also move this after the add_new part, you are not adding the new node as seed
you are also not removing the node
I would expect to see `self.target_node.seed = False`, before calling `self.cluster.seed_nodes`

Reply:
Fixed